### PR TITLE
🐛 hover effect on wallet disconnect

### LINKED ISSUE #4589: Hover effect missing on buttons at wallet dialog
It seems to have a hover effect on the change account and border on the light mode 

https://user-images.githubusercontent.com/5887929/210402441-b36f1cf7-f7a0-4968-b7d6-5150ba019b91.mov

### PR DESCRIPTION
**Pull request by petersopko**
**Thank you for your contribution** to the [KodaDot NFT gallery](https://kodadot.xyz).

👇 \_ Let's make a quick check before the contribution.

## PR Type

- [x] Bugfix
- [ ] Feature
- [ ] Refactoring

## Context

- [x] Closes #4589 
- [ ] Requires deployment <>

#### Before submitting pull request, please make sure:

- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried to respect high code quality standards
- [x] I've didn't break any original functionality
- [x] I've posted a screenshot of demonstrated change in this PR

#### Optional

- [ ] I've tested it at </bsx/collection>
- [ ] I've tested PR on mobile
- [ ] I've written unit tests 🧪
- [ ] I've found edge cases

#### Had issue bounty label?

- [x] Fill up your KSM address: [Payout](https://beta.kodadot.xyz/transfer/?target=FNyt7T1xdbhN7dagf7yGYCRJuE3R45VmTCE3tjzy1rKxa7y)

#### Community participation

- [x] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

## Screenshot 📸

- [x] My fix has changed **something** on UI; a screenshot is best to understand changes for others.
image

### CODE REVIEW

**Review by roiLeo — COMMENTED** [review]
would it be possible to use `NeoButton` component instead?

**Comment by exezbcz** [comment]
change account button does not have hover effect in dark mode, can you check that as well? 
image

**Comment by petersopko** [comment]
> would it be possible to use `NeoButton` component instead?

okay

> change account button does not have hover effect in dark mode, can you check that as well?

yes

File: styles/themes/_light.scss [hunk 1]
```diff
@@ -114,6 +114,14 @@ html.light-mode {
...
       }
     }
+    .navbar__sign-out-button {
+      border-color: $black !important;
+      box-shadow: 4px 4px 0px $black !important;
+      color: $black;
+      &:hover {
+        background-color: $k-accentlight !important;
+      }
+    }
```
Comment (roiLeo) [inline styles/themes/_light.scss:124]: since we'll use `NeoButton`, is it stil needed?

**Review by roiLeo — COMMENTED** [review]
Are you sure there is no css style left to remove?

https://user-images.githubusercontent.com/9987732/210728480-af68b744-502d-4be6-838e-bf3b3ffd759c.mov

File: styles/themes/_light.scss (hunk 1 above)
Comment (petersopko) [reply styles/themes/_light.scss:124]: > Are you sure there is no css style left to remove?
> 
>  Enregistrement.de.l.ecran.2023-01-05.a.8.46.44.AM.mov

accidentally pushed, I'm not done, sorry

**Comment by petersopko** [comment]
@roiLeo should be ok now, I've replaced the buttons related to connecting / disconnecting wallet with NeoButtons, if there are some specific differences regarding the size of buttons (connect is now slightly smaller I think), please let's move forward and follow up in the spirit of @yangwao [newest discussion post](https://github.com/kodadot/nft-gallery/discussions/4605)

**Review by roiLeo — APPROVED** [review]
we might have wrong hover border-color on dark-mode inherited from `.o-btn--connect-dropdown` 
Capture d’écran 2023-01-05 à 10 27 16 AM


otherwise code lgtm

**Comment by yangwao** [comment]
```
Conflicting files
libs/ui/src/components/NeoButton/NeoButton.vue
```
@petersopko 👀

**Comment by petersopko** [comment]
> ```
> Conflicting files
> libs/ui/src/components/NeoButton/NeoButton.vue
> ```
> 
> @petersopko 👀

🚀

**Comment by yangwao** [comment]
pay 30 usd

**Comment by petersopko** [comment]
> pay 30 usd

this didn't work 😅

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $30 @ 24.74 USD/KSM ~ 1.213 $KSM
🧗 [FNyt7T1xdbhN7dagf7yGYCRJuE3R45VmTCE3tjzy1rKxa7y](https://kusama.subscan.io/account/FNyt7T1xdbhN7dagf7yGYCRJuE3R45VmTCE3tjzy1rKxa7y)
🔗 [0x65ae40bc7c86e4de127805017c7596a55a1902e7f0cb5da202185e56205af398](https://kusama.subscan.io/extrinsic/0x65ae40bc7c86e4de127805017c7596a55a1902e7f0cb5da202185e56205af398)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Pull request merged**